docs/book/user-guide/cloud-guide/aws-guide.md

```diff
@@ -0,0 +1,201 @@
+
+{% hint style="warning" %} Keep the access keys secure and do not share them publicly as they allow full access to your AWS resources! {% endhint %}
+
+3. If you have not already, install the AWS and S3 ZenML integrations:
+
+```shell
+zenml integration install aws s3 -y
+```
+
+## 2) Create a Service Connector within ZenML
+
+Create an AWS Service Connector within ZenML the service connector will allow ZenML and other ZenML components to authenticate themselves with AWS.
+
+{% tabs %}
+{% tab title="CLI" %}
+
+```shell
+zenml service-connector register aws_connector \
+  --type aws \
+  --auth-method secret-key \
+  --aws_access_key_id=<YOUR_ACCESS_KEY_ID> \  
+  --aws_secret_access_key=<YOUR_SECRET_ACCESS_KEY> \
+  --region=<YOUR_REGION>
+```
+
+Replace <YOUR_ACCESS_KEY_ID>, <YOUR_SECRET_ACCESS_KEY> and <YOUR_REGION> with the respective values.
+{% endtab %}
+{% tab title="Dashboard" %}
+<figure><img src="../../.gitbook/assets/aws-service-connector-type.png" alt=""><figcaption><p>Create the service connector easily in the dashboard</p></figcaption></figure>
+
+You can use the "Connectors" section of the ZenML dashboard to create a AWS service connector in a no-code manner.
+{% endtab %}
+{% endtabs %}
+
+## 3) Create Stack Components
+
+### Artifact Store (S3)
+
+An [artifact store](../production-guide/remote-storage.md) is used for storing and versioning data flowing through your pipelines.
+
+1. Before you run anything within the ZenML CLI, head on over to AWS and create a AWS bucket, in case you don't already have one that you can use. 
+
+```shell
+aws s3api create-bucket --bucket your-bucket-name --region your-region-name
+```
+
+Once this is done, you can create the ZenML stack component as follows:
+
+2. Register an S3 Artifact Store with the connector
+
+```shell
+zenml artifact-store register cloud_artifact_store -f s3 --path=s3://bucket-name --connector aws_connector
+```
+
+More details [here](../../stacks-and-components/component-guide/artifact-stores/s3.md).
+
+### Orchestrator (Sagemaker Pipelines)
+
+An [orchestrator](../production-guide/cloud-orchestration.md) is the compute backend to run your pipelines.
+
+1. Before you run anything within the ZenML CLI, head on over to AWS and create a Sagemaker domain (Skip this if you already have one)
+
+```shell
+aws sagemaker create-domain --region <REGION> --domain-name <DOMAIN_NAME>
+```
+
+Once this is done, you can create the ZenML stack component as follows:
+
+2. Register a Sagemaker Pipelines orchestrator stack component:
+
+```shell
+zenml orchestrator register sagemaker-orchestrator --flavor=sagemaker --region=<REGION>
+```
+
+**Note**: SageMaker orchestrator utilizes the AWS configuration for operation and does not require direct connection via a service connector for authentication, as it relies on your AWS CLI configurations or environment variables.
+
+More details [here](../../stacks-and-components/component-guide/orchestrators/sagemaker.md).
```

Comment:
We probably want a little info box explaining exactly what the choices here determine -- since each time you create these domains, I think you're incurring costs and they're hosting a notebook etc which also costs money. So maybe just help better inform people about the Sagemaker way (which we need to figure out, as we already discussed)